analytics.js

```diff
@@ -936,9 +936,9 @@ class Analytics {
     this.storage.clear(flag);
   }
 
-  getAnonymousId() {
+  getAnonymousId(autoAssignAnonId) {
```

Comment:
The TS declaration file updates are missing.

Reply:
If we update the TS declaration then it will be publicly available. Anyone can call
rudderanalytics. getAnonymousId("segment");
to fetch segment's anonymous id.

Reply:
Because of the change in the signature, it'll be a breaking change for the existing users. Let the users pass the same anonymous ID options object here which would anyway be an optional argument.

Reply:
Addressed